Expensify/App
# Polyfill `Array.prototype.toSorted`

### Details

This is a follow-up to https://github.com/Expensify/App/pull/34257. The `toSorted` implementation is not yet available on non-Web platforms. This PR removes the usage of that function.

### Fixed Issues

$ https://github.com/Expensify/App/issues/32001
PROPOSAL:

### Tests

Same as "QA Steps"

### Offline tests

### QA Steps

- Precondition: you have assigned cards, and they are shown on the Wallet page
- Go to the Wallet page
- Verify the page is displayed correctly

### PR Author Checklist


### Screenshots/Videos
<details>
<summary>Android: Native</summary>

</details>

<details>
<summary>Android: mWeb Chrome</summary>

</details>

<details>
<summary>iOS: Native</summary>

</details>

<details>
<summary>iOS: mWeb Safari</summary>

</details>

<details>
<summary>MacOS: Chrome / Safari</summary>

</details>

<details>
<summary>MacOS: Desktop</summary>

</details>
